layouts/partials/scripts.html

```diff
@@ -6,12 +6,16 @@
     Concatenate, minify, and fingerprint scripts. Generated file will contain content
     hash in filename, so no need to add query param to cache bust.
 -->
-{{ $s := slice (resources.Get "js/main.js") }}
-{{ $s = $s | append (resources.Get "js/nav.js") }}
-{{ $s = $s | append (resources.Get "js/carousel.js") }}
-{{ $s = $s | append (resources.Get "js/langchoose.js") }}
-{{ $s = $s | append (resources.Get "js/noselect.js") }}
-{{ $s = $s | append (readFile "node_modules/clipboard-polyfill/build/clipboard-polyfill.promise.js" | resources.FromString "js/node/clipboard-polyfill.js") }}
-{{ $s = $s | append (resources.Get "js/copybutton.js") }}
-{{ $js := $s | resources.Concat "js/bundle.js" | resources.Minify | resources.Fingerprint }}
-<script src="{{ $js.RelPermalink }}" integrity="{{ $js.Data.Integrity }}"></script>
+{{ if fileExists "node_modules" }}
```

Comment:
Actually, thinking about it more just now, I should check for the actual `clipboard-polyfill` file here, as it's possible for folks to have `node_modules` currently, but haven't run `make ensure` again to get the new `clipboard-polyfill` dependency. Will fix.